### ISSUE TITLE
Add note about cache version incompatibility to NWIS Client docs

### ISSUE THREAD

**Issue by jarq6c** [NOAA-OWP/hydrotools#193]
Owing to #192 , it may behoove us to add a little note to the docs recommending against attempting to re-use caches after upgrading client versions.

**Comment by jarq6c** [comment]
Thanks @hellkite500 for the suggestion.

**Comment by aaraney** [comment]
Yeah, I agree. I think this would be a helpful addition to what could be an FAQ section in the `nwiw_client` docs. Having thought about it, I am hesitant to add a logic into the code base to check for this case, so I think docs are the way to go. Ill try to remember to open a PR to get this added in sometime this week.

**Comment by jarq6c** [comment]
A new `hydrotools.waterdata_client` is under development. This issue will probably close after the beta.

The new client uses `diskcache` which will hopefully allow more flexibility.

**Comment by jarq6c** [comment]
Closed by #318
